### PR TITLE
Simple Notification Modal Improvement

### PR DESCRIPTION
**Pull request by dstrukt**
Simple changes:
- Adjusts border radius
- Changes background-color to take advantage of the hover (currently background is the same as the hover state color .. might have originally been in the mocks, but correcting)
- Adds border to help differentiate from background

Screen Shot 2022-05-26 at 7 18 20 AM

### CODE REVIEW

**Comment by NicolasDorier** [comment]
it fails because you forgot to separate the numbers with comma

**Comment by dstrukt** [comment]
Ahhh good catch, not sure how I missed those / what I was thinking, will update and rebase.

**Comment by dstrukt** [comment]
Should be fixed now.

File: BTCPayServer/wwwroot/main/layout.css [hunk 1]
```diff
@@ -296,12 +296,13 @@
 
 #NotificationsDropdown {
     border: 0;
-    border-radius: 4px;
-    background: var(--btcpay-bg-tile);
-    box-shadow: 0 2px 16px rgba(0, 0, 0, 0.08);
+    border-radius: 8px;
```
Comment (dennisreimann) [inline BTCPayServer/wwwroot/main/layout.css:299]:
```suggestion
    border-radius: var(--btcpay-border-radius-l);
```

**Review by dennisreimann — APPROVED** [review]

**Comment by dstrukt** [comment]
Good call on that last variable change, need to remember to do that myself!

**Pull request merged**